### PR TITLE
Audit Api Created. Closes #1.2

### PR DESCRIPTION
**Pull request by gopal-mahajan**

### CODE REVIEW

**Comment by gopal-mahajan** [comment]
[Newman Summary Report.pdf](https://github.com/datakaveri/iudx-resource-server/files/7641569/Newman.Summary.Report.pdf)
[Surefire Report.pdf](https://github.com/datakaveri/iudx-resource-server/files/7641617/Surefire.Report.pdf)

**Comment by gopal-mahajan** [comment]
[Surefire Report.pdf](https://github.com/datakaveri/iudx-resource-server/files/7659420/Surefire.Report.pdf)

Updated Surefire Report.

File: src/main/java/iudx/resource/server/apiserver/handlers/AuthHandler.java [hunk 1]
```diff
@@ -178,10 +177,13 @@ private JsonObject generateResponse(ResponseUrn urn, HttpStatusCode statusCode)
...
 
-
   /**
-   * extract id from request (path/query or body )
-   * 
+   * <<<<<<< HEAD extract id from request (path/query or body )
+   *
+   * <p>======= extract id from path param
+   *
+   * <p>>>>>>>> Audit Api Created.
```
[the remark this answers is not in the document]
Comment (gopal-mahajan) [reply src/main/java/iudx/resource/server/apiserver/handlers/AuthHandler.java]: done.

**Review by kailash — APPROVED** [review]
LGTM

**Pull request merged**